Advanced-Recruit-Training/src/BatteryStateDisplayService.cpp

```diff
@@ -4,7 +4,7 @@
 #include "BatteryStateDisplayService.h"
 #include "I_BatteryDataSource.h"
 #include "I_BatteryStateOfChargeService.h"
-
+#include "BatteryStateOfChargeService.h"
```

Comment:
Shouldn't be necessary, you would only need this header if you were instantiating it here, but you only need the interface type `I_BatteryStateOfChargeService`